python/MooseDocs/extensions/listing.py

```diff
@@ -403,7 +403,7 @@ def extractInputBlocks(filename, blocks):
         hit = pyhit.load(filename)
         out = []
         for block in blocks.split():
-            node = moosetree.find(hit, lambda n: n.fullpath.endswith(block.strip('/')))
+            node = moosetree.find(hit, lambda n: n.fullpath.endswith(block.rstrip('/')))
```

Comment:
I am fine with this extension adjustment. 👍🏻 Will leave other aspects to @lindsayad since he is more familiar with the MFEM integration.